src/Console/Commands/BroadcastFaker/Fake.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+
+namespace Redbeed\OpenOverlay\Console\Commands\BroadcastFaker;
+
+
+use Illuminate\Support\Arr;
+
+class Fake
```

Comment:
Wouldnt it be able to solve this with PHP Faker? This would make generating further fake test data easier, but might be more effort right now since you've solved it this way already
https://github.com/fzaninotto/Faker

Reply:
Not really because:
- PHP-Faker is not active anymore 
- This Fake Data/Repo should not be overloaded by a big package for only "usernames",